### PR TITLE
EZP-27992: Display the name and the email address separately in Content view for Author field

### PR DESCRIPTION
**Pull request by adamwojs**
> JIRA: https://jira.ez.no/browse/EZP-27992

# Description
This PR contains patch for the EZP-27992: "Content view for Author field - Display the name and the email address separately"

Notes for reviewers:
* Original JIRA ticket don't take into consideration that editor can specify multiple authors
* Removed redundant inheritance from EzPublishCoreBundle, because it causes infinite recursion if `content fields.html.twig` extends `@EzPublishCore/content_fields.html.twig`

### CODE REVIEW

**Comment by Nattfarinn** [comment]
@webhdx is off till 16th. @alongosz. can you do a quick Code Review?

File: src/bundle/Resources/config/views.yml [hunk 1]
```diff
@@ -1,5 +1,7 @@
 system:
     admin:
+        user:
+            login_template: "@EzPlatformAdminUi/Security/login.html.twig"
```
Comment (alongosz) [inline src/bundle/Resources/config/views.yml]: @adamwojs  How is this related to this PR?

Comment (adamwojs) [reply src/bundle/Resources/config/views.yml]: I've removed bundle inheritance, so if we still would like to have custom login page for administration panel we need those config lines
